docs/source/schema_models.rst

```diff
@@ -158,42 +158,42 @@ with pandera. Pydantic should be used to check any built-in types.
 
 .. testcode:: union_dataframe_built_in_types
 
-    from typing import Union
-    import pandas as pd
-    import pandera as pa
-    from pandera.typing import DataFrame, Series
+from typing import Union
```

Comment:
@kr-hansen why de-indent this? It won't render properly: https://pandera--995.org.readthedocs.build/en/995/schema_models.html#validate-against-multiple-schemas

Reply:
@cosmicBboy Good catch.  My mistake while re-running those tests.  Fixed in [30188b5](https://github.com/unionai-oss/pandera/pull/995/commits/30188b5136b17679cdfb73b795bb218f392491db)